docs/README.md

```diff
@@ -2,7 +2,19 @@
 
 ![](./assets/banner-search.png)
 
-The Search module provides a mirrored index of all CMS content that is optimized for search relevance, speed and accuracy. The Search module overrides the default search functionality to query the specialized search index in place of a standard MySQL query. This means all default search operations using the CMS search APIs for posts such as `WP_Query`, `get_posts`, `get_search_form()` and the REST API search endpoints will transparently make use of the search index.
+The Search module provides a mirrored Elasticsearch index of all CMS content that is optimized for search relevance, speed and accuracy. This operates at a higher level than the primary datastore (MySQL) and is the recommended solution for searching.
+
+The benefits of using Elasticsearch over MySQL search queries include:
+
+- Speed
+- Advanced text analysis
+  - Removing HTML
+  - Language dependent word stemming and stopwords
+  - Language dependent tokenisation
+- Configurable relevancy scores
+- Fuzzy matching
+- Synonyms
+- Aggregations for faceted search and statistics
 
 The default Search index and related functionality is provided by the [ElasticPress plugin](https://github.com/10up/ElasticPress) and the multilingual support is derived from [the WordPress.com ElasticSearch library](https://github.com/Automattic/wpes-lib).
 
```

Comment:
Not sure what you mean here about hierarchy?

Reply:
This was referring to the example config block which is actually below this line, but I couldn't suggest changes on it because it didn't actually change.

Reply:
Ah gotcha - I think you can expand the preview and select multiple lines. Lets spin up a platform-dev issue. Maybe we need to write an RFC for documentation standards that goes into real detail around config and code examples etc.

Reply:
Yep, it's definitely a broader conversation 👍